src/Mod/Draft/DraftEdit.py

```diff
@@ -37,7 +37,7 @@
     import FreeCADGui
     import DraftTools
     from DraftTrackers import *
-    from PySide import QtCore
+    from PySide import QtCore, QtGui
```

Comment:
Is it possible for you to change the `import *` statement to indicate only the necessary functions?

I mean
```py
from DraftTrackers import circlertracker, rectangletracler, splinetracker
```
This is better in the long run, as we'll know what is used exactly and what is not used, and thus how we can split the `DraftTrackers` and the `Draft_Edit` command into more modules.

Reply:
done, i took the chance to do the same for import Draft.
I also discovered that i needed to import coin !

Reply:
This is the reason importing everything is bad, because you can't tell what is actually needed.